File: content/learn/arrays-easy/03-is-sorted/solution.js
function isSorted(a) {
  for (let i = 1; i < a.length; i++) {
    if (a[i - 1] > a[i]) return false;
  }
  return true;
}

const n = parseInt(readline(), 10);
const a = n > 0 ? readline().split(" ").map(Number) : [];
print(isSorted(a) ? "true" : "false");

File: content/learn/arrays-easy/03-is-sorted/solution.py
def is_sorted(a):
    for i in range(1, len(a)):
        if a[i - 1] > a[i]:
            return False
    return True


n = int(input())
a = list(map(int, input().split())) if n > 0 else []
print("true" if is_sorted(a) else "false")
